### PR TITLE
update max-age for assets

### PR DESCRIPTION
**Pull request by fredsmith**

### CODE REVIEW

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 [hunk 1]
```diff
@@ -97,7 +97,7 @@ server {
 
   location / {
     location ~ \.(js|css)$ {
-      add_header 'Cache-Control' 'public, max-age=31536000, immutable';
+      add_header 'Cache-Control' 'public, max-age=3600, immutable';
```
Comment (albemarle) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: This one, I don't think we need to change, it really is immutable.

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 [hunk 2]
```diff
@@ -174,7 +174,11 @@ server {
   }
 
   location /static/ {
-    add_header 'Cache-Control' 'public, max-age=31536000, immutable';
+    add_header 'Cache-Control' 'public, max-age=3600, immutable';
```
Comment (albemarle) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: Let's take off immutable-- our permalinks might be here, which are exactly the opposite of immutable (same URL, new content).

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 [hunk 3]
```diff
@@ -174,7 +174,11 @@ server {
...
+  }
+
+  location /images/ {
+    add_header 'Cache-Control' 'public, max-age=3600, immutable';
```
Comment (albemarle) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: Let's take off the immutable.

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 [hunk 4]
```diff
@@ -215,7 +219,7 @@ server {
 
   location / {
     location ~ \.(js|css)$ {
-      add_header 'Cache-Control' 'public, max-age=31536000, immutable';
+      add_header 'Cache-Control' 'public, max-age=3600, immutable';
```
Comment (albemarle) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: How is this grouping different than the part that starts on line 98?

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 [hunk 5]
```diff
@@ -215,7 +219,7 @@ server {
...
       try_files $uri @drupal;
```
Comment (albemarle) [inline playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: This seems unlikely to do much.  :P

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 (hunk 4 above)
Comment (fredsmith) [reply playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: it's in a different `server {}` block, although I don't know why there are two of them defined.

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 (hunk 5 above)
Comment (fredsmith) [reply playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: I'm happy to yank out all the drupal stuff, but maybe in a different PR.

**Review by albemarle — APPROVED** [review]

File: playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2 (hunk 2 above)
Comment (albemarle) [reply playbooks/roles/nginx/templates/edx/app/nginx/sites-available/prospectus.j2]: Update, our permalinks are not necessarily there, but can have any URL.  I can't imagine any of them end in .js or .css, though.  (Hopefully not famous last words...) All the more reason not to have anything else marked immutable.

**Pull request merged**